charms/kfp-api/charmcraft.yaml

```diff
@@ -11,3 +11,7 @@ bases:
 parts:
   charm:
     charm-python-packages: [setuptools, pip]  # Fixes install of some packages
+    # Install jinja2 (a dependency of charmed-kubeflow-chisme) from binary to avoid build-time issues
+    # See https://github.com/canonical/bundle-kubeflow/issues/883
+    # Remove when https://github.com/canonical/charmcraft/issues/1664 is fixed
+    charm-binary-python-packages: [jinja2]
```

Comment:
I wonder if this should be left unpinned, or if it should be pinned to the same thing as in requirements.txt.  

Did some local testing and I saw we currently build the charm by downloading the binary for jinja=3.1.2, which is **not** the latest but **is** the version specified in requirements.txt.  So I think somehow charmcraft is resolving this unpinned version by using the requirements.txt?

I'm asking more about it [here](https://matrix.to/#/!LhFxJIPEcCacgdMghH:ubuntu.com/$JzNRrFJYLfxvR7C3XRFA18uAWpnrfiGLQjIEzmPFs4o?via=ubuntu.com&via=matrix.org), but probably leaving it unpinned here for now is ok